lz4/Makefile

```diff
@@ -0,0 +1,60 @@
+CC = gcc
+CFLAGS = --std=c99 -O0 -g -Wall -Wextra -I../PIM-common/common/include
+DPU_OPTS = `dpu-pkg-config --cflags --libs dpu`
+
+# define DEBUG in the source if we are debugging
+ifeq ($(DEBUG), 1)
+	CFLAGS+=-DDEBUG
+endif
+
+# Default Parameters
+NR_DPUS = 1
+NR_TASKLETS = 1
+
+SOURCE = dpu_lz4.c lz4_compress.c lz4_decompress.c
+
+.PHONY: default all dpu host clean tags
+
+default: all
+
+all: dpu host
+
+clean:
+	$(RM) dpu_lz4
+	$(MAKE) -C dpu-decompress $@
+	$(MAKE) -C dpu-compress $@
+
+dpu:
+	DEBUG=$(DEBUG) NR_DPUS=$(NR_DPUS) NR_TASKLETS=$(NR_TASKLETS) $(MAKE) -C dpu-decompress
```

Comment:
Eventually we will need compression & decompression in the same DPU image
This is not necessary for the initial commit though